setup.py

```diff
@@ -98,7 +98,7 @@
         "pillow>=6.1",
         "pyarrow>=5.0.0",
         "pyscal>=0.7.5",
-        "scipy>=1.2",
+        "scipy>=1.2,<1.9.3",
```

Comment:
Remove this now since it is an upstream issue, @lindjoha moved it into CI instead.